src/index.json

```diff
@@ -1232,6 +1232,53 @@
                 "sha256Digest": "df397272396c684972d1185e16439159427795b305f67e47fc37447a0c4d1257"
             }
         ],
+        "deploy-to-azure": [ 
```

Comment:
is this extension for github action only? can user use it as a normal extension at local console?

Reply:
I didn't get your question.
 This extension is to Deploy your app to Azure[supports aks, aci, functionapp as of now] via creating/configuring your Github Actions workflow.

Reply:
that answered my question. just to confirm whether user could `az aks app up` as a normal cli command outside of GitHub action. seems not.

Reply:
What do you mean outside of GitHub Actions?

These commands are basically targeting users who want to setup actions workflow for their repo and deploy their app to Azure.
It just abstracts away the concepts of Devops, cwires up and actions workflow yml for the app, and makes it easier for user to have his first pipeline up ad running.

Reply:
there're similar up command in cli today like `az webapp up` it will create/deploy webapp for user. For this `aks app up` command, it's used to automate the GitHub action workflows of aks/aci/function as mentioned above. Actually I think this command is really good convenient in automation scenario.

Reply:
Yes, you are right. This command is similar to az webapp up command.